src/main/java/ejercicio/banco/repository/MySqlAccountRepository.java

```diff
@@ -49,22 +49,22 @@ public Account find(int id) {
         Connection con = dataBaseUtil.startConnection();
 
         try {
-            PreparedStatement pstmt = con.prepareStatement("select * from ACCOUNTS where id = " + id);
+            PreparedStatement pstmt = con.prepareStatement("select * from ACCOUNTS where id=" + id);
             ResultSet rs = pstmt.executeQuery();
-            if (rs.next()) {
+            if(rs.getInt("id")== id) {
```

Comment:
Can you explain why this? This could never be false. I mean, if you query is `select * from ACCOUNTS where id=1`, it could either return that entry or nothing, but never other account :D

Reply:
Changed back in "repository tests fixed" commit.